src/setup.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import setuptools	
+import os	
+
+readme_path = os.path.join("..", "README.md")	
+with open(readme_path, "r") as f:	
+    long_description = f.read()	
+
+
+setuptools.setup(	
+    name="pl_orquestra",	
```

Comment:
`pennylane_orquestra`?

Reply:
Good point! Went with `pennylane_orquestra_steps` for now just so that the packages installed with `pennylane_orquestra/setup.py` and the `pennylane_orquestra/src/setup.py` files have separate names. The idea is that this inner `setup.py` is used by Orquestra when executing a step from a workflow. Lmk if another name would be preferred.

Reply:
A bit late, but good decision @antalszava. Having both `pennylane_orquestra` and `pl_orquestra` was a bit ambiguous